src/types.ts

```diff
@@ -0,0 +1 @@
+export type ReviewComment = { body: string; path: string; line: number };
```

Comment:
Consider using a more descriptive name for the type 'ReviewComment' to clarify its purpose in the context of the application. This can help improve code readability and maintainability.